### ISSUE TITLE
[Max] Possible to export Xref Objects

### ISSUE THREAD

**Issue by Remi-Tribia** [BabylonJS/Exporters#362]
Hi,

I'm using multiple assets separated in their own .max files and then assemble a bigger scene using Xref Objects, placing them and baking lightmaps for the entire scene. The lightmap uses UV2, all the differents xref objects are laid out in the same UV space. Since the UV2 is unwrapped in the big scene I would need to export all those Xref Objects, as if it was classic objects.

Don't know if it's technically possible but it would be awesome, it would prevent me from merging all the objects in the scene each time I want to export it.

Cheers!

**Comment by kcoley** [comment]
@Remi-Tribia to verify, are you wanting an option to just export the mesh geometry without any materials and textures?

**Comment by Remi-Tribia** [comment]
Yes, only the objects without any material. I'm using `$.material = undefined` before exporting for now. I export all the materials using another scene to create some kind of material library.

**Comment by kcoley** [comment]
@Remi-Tribia I just merged an update for this, so you can now toggle material export from the 3ds Max exporter window.

**Comment by Remi-Tribia** [comment]
Thanks a lot! I will test it asap.
